R/MeasureSurvGraf.R

```diff
@@ -112,7 +112,7 @@ MeasureSurvGraf = R6::R6Class("MeasureSurvGraf",
 
       score = weighted_survival_score("graf",
         truth = prediction$truth,
-        distribution = prediction$data$distr, times = ps$times,
+        distribution = prediction$distr, times = ps$times,
```

Comment:
```suggestion
        distribution = prediction$distr, times = ps$times, FIXME: this needs to be reverted, I'll check this
```

Reply:
So what happened here, is that `prediction$data$distr` is a survival array object so later in the code `weighted_survival_score` does some things that only work on matrices and not arrays. By passing an `Arrdist` in this case, code works

Reply:
I really think this might break parametric models

Reply:
And it's then inconsistent with all other measures